src/main/java/com/amazon/opendistroforelasticsearch/security/auditlog/config/AuditConfig.java

```diff
@@ -385,4 +383,10 @@ public String toString() {
                 .filter(settings::hasValue)
                 .collect(Collectors.toSet());
     }
+
+    public static final Set<String> FIELD_PATHS = Sets.union(
+            DefaultObjectMapper.getFields(AuditConfig.class, "/"),
+            Sets.union(
+                    DefaultObjectMapper.getFields(AuditConfig.Filter.class, "/audit/"),
```

Comment:
reuse `AuditConfig.Filter.FIELDS` and `ComplianceConfig.FIELDS`.

Reply:
A prefix path needs to be added to represent the resource path hence did not reuse

Reply:
Done